msix/UbuntuProForWSL/Package.appxmanifest

```diff
@@ -30,61 +15,43 @@
       </virtualization:ExcludedKeys>
     </virtualization:RegistryWriteVirtualization>
   </Properties>
-
   <Dependencies>
     <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
     <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.14393.0" MaxVersionTested="10.0.14393.0" />
     <PackageDependency Name="Microsoft.VCLibs.140.00.UWPDesktop" MinVersion="14.0.24217.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
   </Dependencies>
-
   <Resources>
-    <Resource Language="x-generate"/>
+    <Resource Language="x-generate" />
```

Comment:
yeah, I think you did run some automatter, no? I have no issue here, but on the ones below, I found the xml way more readable before the commit.

Reply:
Yeah, since building the Appx reformats the manifest, I decided to commit the autoformatted file. This way the diff after building is just the version number.